src/JustEat.StatsD.Tests/SocketTransportTests.cs

```diff
@@ -35,16 +35,21 @@ public static void SocketTransportCanSendOverIPWithoutError()
         [Fact]
         public static void NullEndpointSourceThrowsInConstructor()
         {
-            Should.Throw<ArgumentNullException>(
+            Assert.Throws<ArgumentNullException>(
+                "endPointSource",
                 () => new SocketTransport(null, SocketProtocol.IP));
-
         }
 
         [Fact]
         public static void InvalidSocketProtocolThrowsInConstructor()
         {
-            Should.Throw<ArgumentOutOfRangeException>(
-                () => new SocketTransport(LocalStatsEndpoint(), (SocketProtocol)42));
+            SocketProtocol socketProtocol = (SocketProtocol)42;
```

Comment:
Can you use `var` here? and name it `invalidSocketProtocol` ?

Reply:
`var` is fine, but I prefer to have the variable and argument names match for simple tests scenarios like this. Happy to change it if we really want the name like that though.